server/routes/api/article.js

```diff
@@ -17,5 +18,6 @@ router.get('/articles/:slug', ArticleController.getSingleArticle);
 router.delete('/articles/:slug', checkToken, ArticleController.deleteArticle);
 router.put('/articles/:slug', checkToken, upload.single('imgUrl'), ArticleValidations.validateUpdateArticle, ArticleController.updateArticle);
 router.post('/articles/:slug/rate', checkToken, RateValidations.validateRating, RateController.rateArticle);
+router.post('/articles/:slug/:likeType', checkToken, LikeController.likeArticle);
```

Comment:
I think `:likeType'` can be a query parameter instead. e.g `?query=value&query=value` @samdiano